robot-agent/mqtt.js

```diff
@@ -215,6 +215,74 @@ const staticInfo = () => {
   });
 };
 
+// list of self checks to run
+const selfChecks = [
```

Comment:
Since all these use an execSync (as intended), I think we should make this declarative, not procedural. Meaning that `selfChecks` should just list the commands, a function to evaluate the result. Also, it should be an object (with name as key), not an array. This way we'll be able to look up the tests from the front-end.